theano/gpuarray/opt.py

```diff
@@ -55,14 +59,34 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
 gpu_cut_copies = EquilibriumDB()
 
+old_not_transferred = []
+new_not_transferred = []
```

Comment:
I've removed them in the recent commit. Actually, do i need them ? They were kept for knowing the nodes that aren't transferred in the new optimizer, but were in the old optimizer.